src/components/home/TopicDrawer.js

```diff
@@ -0,0 +1,163 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { compose } from 'redux';
+import PropTypes from 'prop-types';
+
+import List from '@material-ui/core/List';
+import ListItem from '@material-ui/core/ListItem';
+import ListItemText from '@material-ui/core/ListItemText';
+import MenuList from '@material-ui/core/MenuList';
+import Paper from '@material-ui/core/Paper';
+import Divider from '@material-ui/core/Divider';
+import { withStyles } from '@material-ui/core';
+import Hidden from '@material-ui/core/Hidden';
+import ExpandLess from '@material-ui/icons/ExpandLess';
+import ExpandMore from '@material-ui/icons/ExpandMore';
+import Collapse from '@material-ui/core/Collapse';
+import Typography from '@material-ui/core/Typography';
+
+import { searchEnvironmentsByTopic, resetSearchedEnvironments } from '../../actions/environments';
+import About from './About';
+
+const drawerWidth = 240;
+
+const styles = theme => ({
+  root: {
+    flexGrow: 1,
+    backgroundColor: 'AliceBlue',
+  },
+  title: {
+    margin: theme.spacing.unit * 2,
+    marginTop: theme.spacing.unit * 6,
+  },
+  drawerPaper: {
+    width: drawerWidth,
+    flexShrink: 0,
+    paddingTop: '0',
+    paddingBottom: '0',
+  },
+  wrapper: {
+    display: 'flex',
+    flexFlow: 'row nowrap',
+  },
+  gridStyle: {
+    [theme.breakpoints.up('lg')]: {
+      width: '100%',
+    },
+  },
+});
+
+class TopicDrawer extends Component {
+  constructor(props) {
+    super(props);
+    const topTopics = this.props.topics.filter(topic => !topic.parentTopic);
+    this.state = {
+      open: Array.apply(null, Array(topTopics.length)).map(function() {
+        return false;
+      }),
+      error: '',
+    };
+  }
+
+  componentDidUpdate(prevProps) {
+    if (this.props.topics !== prevProps.topics) {
+      const topTopics = this.props.topics.filter(topic => !topic.parentTopic);
```

Comment:
I might rename this to `parentTopics` to make it more apparent that this `top` is a hierarchy, not top as in most popular.